src/Uno.Core.Extensions.Collections/Uno.Core.Extensions.Collections.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras" ToolsVersion="15.0">
 	<PropertyGroup>
-		<TargetFrameworks>uap10.0.17763;uap10.0.19041;net46;netstandard2.0;netstandard2.1;net5.0;net6.0</TargetFrameworks>
+		<TargetFrameworks>uap10.0.17763;uap10.0.19041;net46;netstandard2.0;netstandard2.1;net7.0</TargetFrameworks>
```

Comment:
It would be a binary breaking change for apps that transitively use those versions.

Reply:
I think it can only happen if an API in .NET itself broke binary compat for some API we use? So I feel it's quite unlikely, but added net5.0;net6.0 back anyway.